lib/Symbols.js

```diff
@@ -47,7 +47,9 @@ Symbols.getLastSymbolTerm = function(node, symbolName) {
 // e.g. 4x^2 + 2x + y with `symbolName=x` would return y
 Symbols.getLastNonSymbolTerm = function(node, symbolName) {
   if (isSymbolTerm(node, symbolName)) {
-    return new Node.PolynomialTerm(node).getCoeffNode();
+    if (Node.PolynomialTerm.isPolynomialTerm(node)) {
```

Comment:
That would actually change the behavior of the function, but I've cleaned it up to be more obvious